tests/includes/rest/class-test-outbox-controller.php

```diff
@@ -698,7 +698,7 @@ public function test_get_items_meta_query_for_non_privileged_users() {
 		$this->assertEquals( 200, $response->get_status() );
 		$this->assertSame( 1, (int) $data['totalItems'] );
 		$this->assertCount( 1, $data['orderedItems'] );
-		$this->assertSame( 'https://example.org/activity/1', $data['orderedItems'][0]['object']['id'] );
+		$this->assertSame( 'https://example.org/note/1', $data['orderedItems'][0]['object']['id'] );
```

Comment:
@pfefferle This is still changed…

Reply:
try to find out if it is a bug now or was before... 🤔

Reply:
@obenland this is good to go! the test stores the full `Activity`, so the old code took that and surrounded it with another `Activity`, that is why it returned the `Activity` ID. Now that we do no longer surround the stored `Objects` (`_activitypub_activity_type` is the same as the `Activity` Type), it returns the `Object` ID.

(I hope it was the AI that messed that up in the first place 🫣)